care/emr/management/commands/load_fixtures.py

```diff
@@ -0,0 +1,433 @@
+import json
+import secrets
+from pathlib import Path
+
+import phonenumbers
+from django.conf import settings
+from django.core.management import BaseCommand, call_command
+from django.db import transaction
+from faker import Faker
+from phonenumbers import NumberParseException, PhoneNumberFormat
+
+from care.emr.models import FacilityOrganization, Patient, Questionnaire
+from care.emr.resources.encounter.constants import (
+    ClassChoices,
+    EncounterPriorityChoices,
+    StatusChoices,
+)
+from care.emr.resources.encounter.spec import EncounterCreateSpec
+from care.emr.resources.facility.spec import FacilityCreateSpec
+from care.emr.resources.facility_organization.spec import (
+    FacilityOrganizationTypeChoices,
+    FacilityOrganizationWriteSpec,
+)
+from care.emr.resources.organization.spec import (
+    OrganizationTypeChoices,
+    OrganizationWriteSpec,
+)
+from care.emr.resources.patient.spec import (
+    BloodGroupChoices,
+    GenderChoices,
+    PatientCreateSpec,
+)
+from care.emr.resources.questionnaire.spec import QuestionnaireSpec
+from care.emr.resources.user.spec import UserCreateSpec
+from care.security.models import RoleModel
+from care.users.models import User
+from care.utils.tests.base import CareAPITestBase
+
+
+def generate_unique_indian_phone_number():
+    max_attempts = 1000
+    attempts = 0
+    while True:
+        attempts += 1
+        if attempts > max_attempts:
+            raise RuntimeError(
+                "Failed to generate unique phone number after maximum attempts"
+            )
+        number = 6000000000 + secrets.randbelow(9999999999 - 6000000000 + 1)
+        phone_str = f"+91{number}"
+        try:
+            parsed_number = phonenumbers.parse(phone_str, "IN")
+            if phonenumbers.is_valid_number_for_region(parsed_number, "IN"):
+                result = phonenumbers.format_number(
+                    parsed_number, PhoneNumberFormat.E164
+                )
+                if User.objects.filter(phone_number=result).exists():
+                    continue
+                return result
+        except NumberParseException:
```

Comment:
this function is used for objects other than users, we can move the unique validation to caller 
also we dont need to validate the phone number, just do 
```
return "+91" + random.choice(['9', '8', '7', '6']) + ''.join([str(random.randint(0, 9)) for _ in range(9)])
```